BunqSdk/Exception/ExceptionFactory.cs

```diff
@@ -18,37 +18,49 @@ public class ExceptionFactory
         /// <summary>
         /// Glue to concatenate the error messages.
         /// </summary>
-        private const string GLUE_ERROR_MESSAGES = "\n";
+        private const string SEPARATOR_ERROR_MESSAGES = "\n";
+        
+        /// <summary>
+        /// String format constants.
+        /// </summary>
+        private static string FORMAT_ERROR_MESSAGE = "Response id to help bunq debug: {0}. \n Error message: {1}";
```

Comment:
Probably want to stay away from "forcing" newlines in error messages

Reply:
Why though 🤔 ? Haven't experienced complications with new lines in error messages before.